### PR TITLE
refactor: Remove release 1 feature flags (M2-6906)

### PR DESCRIPTION
**Pull request by sultanofcardio**
### 📝 Description

🔗 [Jira Ticket M2-6906](https://mindlogger.atlassian.net/browse/M2-6906)

This PR removes all references to the `enableMultiInformant` and `enableMultiInformantTakeNow` feature flags. I've also taken the liberty of removing some unused components and related functions. There are probably others that could be removed, and I'd be happy to oblige if you have suggestions.

As part of this ticket I also made two related changes:
- The View Users menu item on the dashboard now points to the participants tab, instead of the applet overview tab
    
- The applet breadcrumb will now navigate to the applet overview tab, instead of the participants tab 
    image

### 📸 Screenshots

N/A

### 🪤 Peer Testing

Sanity checks and regression testing.

Specifically the take now flow

### ✏️ Notes

N/A

### CODE REVIEW

File: src/modules/Dashboard/features/Applets/AppletsTable/AppletItem/AppletItem.tsx [hunk 1]
```diff
@@ -45,13 +44,8 @@ export const AppletItem = ({ item, onPublish }: AppletItemProps) => {
...
   const appletId = item.id;
-  const {
-    featureFlags: { enableMultiInformant },
-  } = useFeatureFlags();
-  const participantPath = generatePath(
-    enableMultiInformant ? page.appletOverview : page.appletRespondents,
-    { appletId },
-  );
+  const overviewPath = generatePath(page.appletOverview, { appletId });
+  const participantPath = generatePath(page.appletParticipants, { appletId });
```
Comment (ChaconC) [inline src/modules/Dashboard/features/Applets/AppletsTable/AppletItem/AppletItem.tsx:48]: "View Users" is still navigating to "Participants" tab.
Based on the PR description, did you mean to update line 130 `viewUsers: () => checkAppletEncryption(() => navigate(participantPath))`

**Review by ChaconC — APPROVED** [review]
Excellent work @sultanofcardio 🗡️ 

Only a minor comment, tested Take Now and general navigation around the app, all seems to work as expected

File: src/modules/Dashboard/features/Applets/AppletsTable/AppletItem/AppletItem.tsx (hunk 1 above)
Comment (sultanofcardio) [reply src/modules/Dashboard/features/Applets/AppletsTable/AppletItem/AppletItem.tsx:48]: Oh I wrote the wrong thing 🤦 It's _supposed_ to go to the participants tab, it used to go to the overview tab. Thanks for pointing this out

**Comment by mbanting** [comment]
@voropa please see the two small nav changes as described in the [description](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1878#issue-2436491835), in case it impacts E2E tests as well.

**Review by mbanting — APPROVED** [review]
Great work @sultanofcardio ! Thank you for cleaning up some of the unused code on the process.

**Pull request merged**